pkg/package-server/apis/operators/packagemanifest.go

```diff
@@ -69,3 +72,21 @@ func descriptionsForAPIServices(apis []operatorsv1alpha1.APIServiceDescription)
 	}
 	return descriptions
 }
+
+// GetRelatedImages returns a list of images listed in Deployments
+func GetRelatedImages(csv *operatorsv1alpha1.ClusterServiceVersion) []string {
```

Comment:
You also want to pick up the images explicitly listed in the CSV, see: https://github.com/operator-framework/operator-registry/blob/7418392774193f088a5f9c4be733da3584de1384/pkg/registry/csv.go#L210